### ISSUE TITLE
SC 3.7 path issue

### ISSUE THREAD

**Issue by dathinaios** [chrisbarrett/sclang-extensions#9]
Hi there,

I am getting an error when trying on 3.7:

```
/usr/local/Cellar/emacs-mac/emacs-24.5-z-mac-5.15/Emacs.app/Contents/MacOS/Emacs: /Applications/SuperCollider/SuperCollider.app/Contents/Resources/sclang: No such file or directory
*** SCLang exited abnormally with code 127 ***

```

In 3.7 `sclang` is in the MacOS folder.

**Comment by chrisbarrett** [comment]
Hey there,

I don't have time to look into this myself, but a PR would be welcome. :)

**Comment by dathinaios** [comment]
No worries. At the moment I can stay at 3.6.6 . I will attempt to fix #6 when I can and then maybe I can have a look. But I have been using Emacs (and thus elisp) for three days so I wouldn't hold my breath :)

Thanks!
